### DISCUSSION TITLE
Project/hardware sanity check, will these devices work?

### DISCUSSION THREAD

**Discussion by bkcberry** [timmbogner/Farm-Data-Relay-System#163] [Q&A] [answered]
Hello all, I just stumbled across FDRS and it sounds like it is perfect for what I am trying to do, which is to use MQTT, via LORA, to switch a remote relay on and off. I don't have any experience with LORA yet so I was hoping someone here would weigh in on these 2 boards and their compatibility with FDRS:

[Will this HiLetgo LORA module work as a gateway?](https://www.amazon.com/gp/product/B07WHRS2XG/ref=ox_sc_act_title_1?smid=A30QSGOJR8LMXA&psc=1)

[And will this MakerFocus LoRa SX1262 Module be suitable as a Node to control the relay?](https://www.amazon.com/gp/product/B07ZHCMFW4/ref=ox_sc_act_title_2?smid=A1N6DLY3NQK2VM&psc=1)

Please and thanks

**Comment by timmbogner** [comment, marked as answer]
Hello!

The first link appears to be the Heltec LoRa32 V3. If this is the case, then it is definitely supported! I have one that I use to troubleshoot support for sx126x LoRa radios. I'll say to be careful handling the OLED display, it is very easy to press and break it. I definitely[ prefer LilyGo products](https://www.lilygo.cc/collections/lora-or-gps) over Heltec, though. I'm hoping to try out the SX1262 variant of their new T3-S3 board soon, but currently I can highly recommend their LoRa32 v1.
Here are the configurations for the Heltec LoRa32 V3:
```
#define RADIOLIB_MODULE SX1262  // ESP32 SX1262 (HELTEC)
#define LORA_SS 8
#define LORA_RST 12
#define LORA_DIO 14
#define LORA_BUSY 13
#define USE_SX126X
```

The second one _should_ work.  I've never tested on the CubeCell boards, but I've compiled the library for it successfully in the past. As always, I'll be happy to help you get it up and running on the CubeCell if you decide to give it a shot.
